src/App.js

```diff
@@ -17,3 +17,6 @@ function App() {
 
```

Comment:
Nice work cleaning this up!

Watch your formatting in your code. It might seem insignificant, but indentation matters for code readability, so on line 11, you'll want to tab that `<h1>` in to be inline with its sibling elements.